### DISCUSSION TITLE
增加pkg-config的支持，还有bin 目录支持

### DISCUSSION THREAD

**Discussion by jingjingxyk** [dixyes/lwmbs#5] [Ideas]
为啥需要这个，解决检测依赖问题

比如检测  libxml2库下面的这个  libxml2-config 

依赖libxml2 库的程序会在 ，预处理阶段会执行下面的程序

```shell

 libxml2-config  --cflags 

```

**Comment by dixyes** [comment]
我看不懂你在说什么。。。

lwmbs有自己的抽象层，不（直接）使用pkgconfig

**Comment by jingjingxyk** [comment]
> 我看不懂你在说什么。。。
> 
> lwmbs有自己的抽象层，不（直接）使用pkgconfig

类似这种 https://github.com/dixyes/lwmbs/discussions/4#discussioncomment-6425719
